storage/sqlite/src/queries.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use storage_core::{DbDesc, DbIndex};
+
+/// Return the prefix of the database key/value tables
+#[inline]
+pub(crate) fn db_table_name(idx: usize) -> String {
```

Comment:
This is a private submodule, all the `pub(crate)` in this module can be just `pub` without exposing them externally as far as I can tell.